tnb/banks.py

```diff
@@ -2,48 +2,79 @@
 
 
 class Bank(BaseClient):
-    def fetch_accounts(self) -> dict:
+    def fetch_accounts(self, page: int = 1, limit: int = 50) -> dict:
```

Comment:
Just my thoughts:
Why not use the offset based pagination ? `offset and limit`
Also `offset` default to begin from 0

Example: We have 100 accounts in the system. To get 10 accounts in places 40 – 49, user can send the following:
```/accounts?limit=10&offset=40```

@shahraizali - What do you think ?

Reply:
> Let user specify offset and limit would be nicer. Assuming user want to a block 1615, they would choose the offset = 1615 and limit = 1 rather than calculating page, pass it to here and recalculate the offset.

Same thoughts.

Reply:
I understand the concern here, but like stated in in the slack channel (and with which I agree with) this is an ease-of-use design decision. Especially for new developers coming in it will be much easier to understand `page` and `limit` system. The `page` argument is pretty straightforward for a new developer.

To elaborate on the examples provided by @sbalasa and @DucPhamTV, these are **easily** achievable with the `page` system and give the exact same results:

@sbalasa
```python
bank.fetch_accounts(page=4, limit=10) 
```

@DucPhamTV
```python
bank.fetch_accounts(page=1615, limit=1) 
```

And for the advanced developers, they can still use the base method `fetch` and add their own request params (such as `limit` and `offset`).